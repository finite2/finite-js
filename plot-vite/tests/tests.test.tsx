import { render } from "./test-utils";

describe("tests work", () => {
  test("yes", () => {
    const { container } = render(<div>Hello world</div>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
