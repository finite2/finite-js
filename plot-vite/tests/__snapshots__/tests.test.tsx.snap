// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`tests work > yes 1`] = `
<div>
  Hello world
</div>
`;
